packages/params-middleware/src/types.js.flow

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+import type { RSAAction } from 'redux-api-middleware';
+
+type AnyAction = {
+  type: string,
+};
+
+export type Action = RSAAction<any, any, any> | AnyAction;
```

Comment:
If you will not go with my suggestion about location of params, then maybe the optional param should be added here to request type?